data/ipl4.cfg

```diff
@@ -22,4 +22,4 @@ IPL_ROOT = %(FILESYSTEM)s/%(name)s
 # add any changes from IPL-3 here (none expected)
 
 [PATHS]
-# no changes from IPL-3 
+sdR_S = $BOSS_SPECTRO_DATA_S/{obs}/{mjd}/sdR-{br}{id}-{frame}.fit
```

Comment:
Do we need an `ipl-4.cfg` file at all, since there will be no IPL-4?  I would prefer to remove this file if we will never be releasing anything from it.  It will also avoid confusion from people if they see it as an option in tree/sdss_access.

Reply:
I will happily delete it, I just wanted to make sure that if it is there it is correct.